sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -56,19 +57,22 @@ def __init__(self, endpoint: str, credential: TokenCredential, **kwargs) -> None
         self._refresh_token = None # type: Optional[str]
         self._expiration_time = 0 # type: float
 
+    @distributed_trace
     def get_acr_access_token(self, challenge: str, **kwargs) -> Optional[str]:
         parsed_challenge = _parse_challenge(challenge)
         refresh_token = self.get_refresh_token(parsed_challenge["service"], **kwargs)
         return self.exchange_refresh_token_for_access_token(
             refresh_token, service=parsed_challenge["service"], scope=parsed_challenge["scope"], **kwargs
         )
 
+    @distributed_trace
     def get_refresh_token(self, service: str, **kwargs) -> str:
         if not self._refresh_token or self._expiration_time - time.time() > 300:
             self._refresh_token = self.exchange_aad_token_for_refresh_token(service, **kwargs)
             self._expiration_time = _parse_exp_time(self._refresh_token)
         return self._refresh_token
 
+    @distributed_trace
```

Comment:
I don't think we need @distributed_trace here.

Reply:
This client is also for internal use only, I think we don't need the decorator in this class and the async one.